devtools/configtsidgen/main.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"flag"
```

Comment:
Is this Go file + the makefile changes + schema changes relevant for converting RequireConfig to Typescript?

Reply:
They are so that the typescript definition of possible config IDs are available for the RequireConfig tools. Alternatively, we could change to an enum in `schema.graphql` but since that file is not auto-generated I put the type generation output in the typescript definition file instead.

We could do the former in a subsequent PR and/or look into having a multi-file schema -- I made the assumption this would be a good compromise -- no API/schema changes, but have the config IDs available for typescript checking.